CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## [v1.10.2](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.10.2) (July 17, 2023)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.10.1...v1.10.2)
+
+**Bug Fixes**
+
+- INTMDB-941: \[Terraform\] Advanced Cluster Doc is not formatted correctly [\#1326](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1326) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- INTMDB-922: mongodbatlas\_event\_trigger is not updated if config\_match is added [\#1305](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1305) ([andreaangiolillo](https://github.com/andreaangiolillo))
+
+**Internal Improvements**
+
+- INTMDB-938: Fix Online Archive tests [\#1321](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1321) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- INTMDB-935: HELP: mongodbatlas\_online\_archive schedule parameter update causing crashing in terraform apply [\#1320](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1320) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- Chore\(deps\): Bump github.com/aws/aws-sdk-go from 1.44.298 to 1.44.299 [\#1312](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1312) ([dependabot[bot]](https://github.com/apps/dependabot))
+
+## [v1.10.2](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.10.2) (July 17, 2023)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.10.1...v1.10.2)
+
+**Bug Fixes**
+
+- INTMDB-941: \[Terraform\] Advanced Cluster Doc is not formatted correctly [\#1326](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1326) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- INTMDB-922: mongodbatlas\_event\_trigger is not updated if config\_match is added [\#1305](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1305) ([andreaangiolillo](https://github.com/andreaangiolillo))
+
+**Internal Improvements**
+
+- INTMDB-938: Fix Online Archive tests [\#1321](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1321) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- INTMDB-935: HELP: mongodbatlas\_online\_archive schedule parameter update causing crashing in terraform apply [\#1320](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1320) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- Chore\(deps\): Bump github.com/aws/aws-sdk-go from 1.44.298 to 1.44.299 [\#1312](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1312) ([dependabot[bot]](https://github.com/apps/dependabot))
```

Comment:
it looks like you run two times the generate changelog here

Reply:
ah I see what you mean, fixing

Reply:
updated